administrator/components/com_users/tmpl/methods/list.php

```diff
@@ -0,0 +1,135 @@
+						<?php  foreach($method['active'] as $record): ?>
+							<div class="com-users-methods-list-method-record d-flex flex-row flex-wrap justify-content-start">
+								<div class="com-users-methods-list-method-record-info flex-grow-1 d-flex flex-column align-items-start gap-1">
+									<?php if ($methodName == 'backupcodes'): ?>
+										<div class="alert alert-info">
+											<h3 class="alert-heading">
+												<span class="icon icon-info-circle icon-info-sign" aria-hidden="true"></span>
+												<?php echo Text::sprintf('COM_USERS_TFA_OTEP_PRINT_PROMPT_HEAD', Route::_('index.php?option=com_users&task=method.edit&id=' . (int) $record->id . ($this->returnURL ? '&returnurl=' . $this->escape(urlencode($this->returnURL)) : '') . '&user_id=' . $this->user->id)) ?>
+											</h3>
+											<p class="text-muted">
+												<?php echo Text::_('COM_USERS_TFA_OTEP_PRINT_PROMPT') ?>
+											</p>
+										</div>
+									<?php else: ?>
+										<div class="com-users-methods-list-method-record-title-container mb-1">
+											<?php if ($record->default): ?>
+												<span id="com-users-methods-list-method-default-badge-small"
+													  class="text-warning me-1 hasTooltip"
+													  title="<?php echo $this->escape(Text::_('COM_USERS_TFA_LIST_DEFAULTTAG')) ?>">
+													<span class="icon icon-star" aria-hidden="true"></span>
+													<span class="visually-hidden"><?php echo $this->escape(Text::_('COM_USERS_TFA_LIST_DEFAULTTAG')) ?></span>
+												</span>
+											<?php endif; ?>
+											<span class="com-users-methods-list-method-record-title fs-4 fw-bold">
+												<?php echo $this->escape($record->title); ?>
+											</span>
+										</div>
+									<?php endif; ?>
+
+									<div class="com-users-methods-list-method-record-lastused my-1 d-flex flex-row flex-wrap justify-content-evenly text-muted w-100">
+										<span class="com-users-methods-list-method-record-createdon w-50">
+											<?php echo Text::sprintf('COM_USERS_TFA_LBL_CREATEDON', $model->formatRelative($record->created_on)) ?>
+										</span>
+										<span class="com-users-methods-list-method-record-lastused-date w-50">
+											<?php echo Text::sprintf('COM_USERS_TFA_LBL_LASTUSED', $model->formatRelative($record->last_used)) ?>
+										</span>
+									</div>
+
+								</div>
+
+								<?php if ($methodName != 'backupcodes'): ?>
+								<div class="com-users-methods-list-method-record-actions my-2 d-flex flex-row flex-wrap justify-content-center align-content-center align-items-start">
+									<a class="com-users-methods-list-method-record-edit btn btn-secondary btn-sm mx-1"
+									   href="<?php echo Route::_('index.php?option=com_users&task=method.edit&id=' . (int) $record->id . ($this->returnURL ? '&returnurl=' . $this->escape(urlencode($this->returnURL)) : '') . '&user_id=' . $this->user->id)?>">
+										<span class="icon icon-pencil"></span>
```

Comment:
```suggestion
										<span class="icon icon-pencil" aria-hidden="true"></span>
```

needs some text it cant be just an icon